apps/jira/jira-app/src/components/App/Steps/InstanceStep.tsx

```diff
@@ -61,51 +38,52 @@ export default class InstanceStep extends React.Component<Props, State> {
         <Heading>Configure</Heading>
         <Paragraph>Select the Jira site and project you want to connect</Paragraph>
         <div className="jira-config" data-test-id="instance-step">
-          <Select
-            testId="instance-selector"
-            className="selector"
-            // @ts-ignore: 2339
-            onChange={(e) => pickResource(e.target.value)}
-            isDisabled={resources.length === 1}
-            value={selectedResource || ''}>
-            <Option value="">Select a site</Option>
-            {resources.map((r) => (
-              <Option key={r.id} value={r.id}>
-                {r.url.replace('https://', '')}
-              </Option>
-            ))}
-          </Select>
+          <div style={{ display: 'flex', gap: 16, alignItems: 'center', width: '100%' }}>
```

Comment:
I see lots of new inline styles being added with the style prop. I think to keep consistency with the existing app code it could be better to add `className` here props instead, and then add styling based on classes to the SCSS file instead.